### ISSUE TITLE
Do relativebasedir and file inclusion work?

### ISSUE THREAD

**Issue by rockyallen** [asciidoctor/asciidoctor-ant#28]
I get the following error message when I try to use relativebasedir, or convert a file that has inclusions (test 2):

````
Render simple_2.ad from /home/rocky/work/asciidoctor-ant-test/doc to /home/rocky/work/asciidoctor-ant-test/dist with backend=docbook
/home/rocky/work/asciidoctor-ant-test/build.xml:25: 
org.jruby.exceptions.RaiseException: (IOError) target directory does not exist: /home/rocky/work/asciidoctor-ant-test/dist
	at RUBY.convert(jar:file:/home/rocky/work/asciidoctor-ant-test/lib/asciidoctor-ant-1.5.0.jar!/gems/asciidoctor-1.5.0/lib/asciidoctor.rb:1450)
	at RUBY.convert_file(jar:file:/home/rocky/work/asciidoctor-ant-test/lib/asciidoctor-ant-1.5.0.jar!/gems/asciidoctor-1.5.0/lib/asciidoctor.rb:1528)
	at RUBY.convertFile(<script>:62)
	at org.jruby.gen.InterfaceImpl942225592.convertFile(org/jruby/gen/InterfaceImpl942225592.gen:13)
BUILD FAILED (total time: 6 seconds)
````

The task works as expected with the same (simple) input file when relativebasedir is false (test 1), so I know the output folder does exist. 
If my file has includes I get the same error irrespective of relativebasedir (test 3/test 4).

System: Ubuntu 12, NetBeans 8.02, ant-1.8.2, asciidoctor-ant-1.5.0.jar. (also tested on Windows 7).

Build script and input files: https://gist.github.com/rockyallen/2416ed16bfb17705a755

**Comment by binout** [comment]
Thanks for the details of the issue, i will try to reproduce ... and fix it.
It seems there is a mismatch between Ant baseDir and Asciidoctor baseDir ...

**Comment by binout** [comment]
Fix in 1.5.1
